great_expectations/checkpoint/checkpoint.py

```diff
@@ -286,7 +289,28 @@ def self_check(self, pretty_print=True) -> dict:
         report_object: dict = {"config": self.config.to_json_dict()}
 
         if pretty_print:
-            print(f"Checkpoint: {self.__class__.__name__}")
+            print(f"\nCheckpoint class name: {self.__class__.__name__}")
+
+        # TODO: <Alex>ALEX</Alex>
+        action_list: Optional[list] = self.config.action_list
+        action_list_present: bool = (
+            action_list is not None
+            and isinstance(action_list, list)
+            and len(action_list) > 0
+        )
+        if not (
+            (
+                self.config.validations
+                and isinstance(self.config.validations, list)
+                and len(self.config.validations) > 0
+            )
+            or action_list_present
+        ):
+            logger.warning(
```

Comment:
I think we should split this into two conditionals, with two different warnings. 
- If the validations attribute is empty, warning should be something like "Your current Checkpoint configuration has an empty or missing validations list - this means you must provide an appropriate validations list when the Checkpoint is run."
- If the actions_list is empty, warning should be something like "Your current Checkpoint configuration has an empty or missing actions_list - this means you must provide an appropriate validations list when the Checkpoint is run, with each validation having it's own defined actions_list."

Reply:
Fixing...